test/spec/provider/zeebe/PriorityDefinitionProps.spec.js

```diff
@@ -153,14 +171,14 @@ describe('provider/zeebe - PriorityDefinitionProps', function() {
     );
 
 
-    it('should create non-existing extension elements and priority definition',
+    it('should create priority definition',
       inject(async function(elementRegistry, selection) {
 
         // given
         const userTask = elementRegistry.get('UserTask_2');
 
         // assume
-        expect(getBusinessObject(userTask).get('extensionElements')).to.not.exist;
+        expect(getExtensionElementsList(getBusinessObject(userTask), 'zeebe:priorityDefinition')[0]).to.not.exist;
```

Comment:
Is this what we want to express?

```suggestion
        expect(getExtensionElementsList(getBusinessObject(userTask), 'zeebe:priorityDefinition')).to.be.empty;
```

Reply:
Yeah this makes more sense.